software/apt-lib.pl

```diff
@@ -457,7 +457,33 @@ sub list_package_repos
 foreach my $f ($sources_list_file, glob("$sources_list_dir/*")) {
 	my $lref = &read_file_lines($f, 1);
 	my $lnum = 0;
+	my (%repo, @types);
+	my $repo_proc = sub {
```

Comment:
I don't understand why this code needs to go into a sub when it's only called once?

Reply:
Fox readability, as it won't fit otherwise into 80 characters in width.

Reply:
AH ok .. I suppose I would have preferred to split this out into a true separate sub, but I suppose it's OK